MultisigTests/Logic/NavigationRouterTests.swift

```diff
@@ -86,6 +86,57 @@ final class ExtendedNavigationRouterTests: XCTestCase {
         XCTAssertEqual(route?.path, "/loadSafe")
     }
 
+    func testLoadSafeWithNetwork() {
+        let url = "https://some.host/new-safe/load?chain=eth"
```

Comment:
Can we extract the base url to constant variable as it's used in all methods? maybe it will be helpful later